rts/lra/jaxrs/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java

```diff
@@ -385,10 +460,24 @@ public void filter(ContainerRequestContext requestContext, ContainerResponseCont
         try {
             if (current != null && isCancel) {
                 try {
-                    lraClient.cancelLRA(current);
+                    // do not attempt to cancel if the request filter tried but failed to start a new LRA
+                    if (progress == null || progressDoesNotContain(progress, ProgressStep.StartFailed)) {
+                        lraClient.cancelLRA(current);
+                        progress = updateProgress(progress, ProgressStep.Ended, null);
```

Comment:
a minor thing, the `ClientLRAFilter` uses the call as `updateProgress(progres, ...)` while this class uses the format `progress = updateProgress(progress, ...`. Would not be better to use the same format of the call (probably just `updateProgress(...)`?

Reply:
This is a server side request filter and it is only on the server that we need to check for coordinator availability. The ClientLRAFilter is for client side processing and is of no concern to the goals of this PR.

Reply:
What other method (that link just takes me to the top of the PR diffs). Is this one another complaint about my coding style not matching up with what you prefer.